### DISCUSSION TITLE
Page is impossible to find

### DISCUSSION THREAD

**Discussion by rossmpersonal** [MicrosoftDocs/office-developer-client-docs#673] [Q&A]
[Enter feedback here]

This page is extremely useful. Especially for development document for Access. However, it is impossible to find when navigating from the Office Developer documentation page, https://docs.microsoft.com/en-us/office/dev/add-ins/ . Please fix this.
---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: e88d1d56-b2a7-0e79-74d6-60f84591c103
* Version Independent ID: a02dcc8f-46eb-8f94-a781-a70d1a6b6c82
* Content: [Office client development](https://docs.microsoft.com/en-us/office/client-developer/office-client-development)
* Content Source: [docs/office-client-development.md](https://github.com/MicrosoftDocs/office-developer-client-docs/blob/master/docs/office-client-development.md)
* Product: **office365**
* GitHub Login: @o365devx
* Microsoft Alias: **o365devx**

**Comment by rossmpersonal** [comment]
Issue still doesn't appear to be fixed. How was the issue fixed? Ticket improperly closed. Please reopen.

**Comment by lindalu-MSFT** [comment]
@rossmpersonal We don't have any resources monitoring or addressing issues in this repo. Therefore, we are closing issues and converting to discussion.

**Comment by rossmpersonal** [comment]
It appears that is the root cause of the problem which must be fixed. The documentation is so bad because no one is addressing issues. What good does discussion do?
